Makefile

```
@@ -109,13 +110,33 @@ run-backfill-embeddings:
 	@if [ ! -f .env ]; then echo "Error: .env file required. Copy .env.example to .env and configure."; exit 1; fi && \
 	(set -a && . ./.env && set +a && go run ./cmd/backfill-embeddings)
 
-# Run the API server (hub-api).
+# Run the full local app: apply River migrations, then run hub-worker in the background and hub-api in the foreground.
 # Config: .env if present, else environment variables; env vars override .env. Copy .env.example to .env or set env vars.
-run:
+run: river-migrate
```

checkmake (0.3.2)

[warning] Target body for "run" exceeds allowed length of 5 lines (17).

(maxbodylength)